apps/snapodds/src/services/tracking/properties/snap-view-snap-failed-properties.ts

```diff
@@ -0,0 +1,21 @@
+import { AnalyticsProperties } from '../analytics-properties';
+
+export interface SnapViewSnapFailedProperties extends AnalyticsProperties {
```

Comment:
Please move all properties to the package above (tracking).

Reply:
done